compose/composer.go

```diff
@@ -1,105 +0,0 @@
-package compose
```

Comment:
do you think we will never need the functionality to generate / dump paylods?

Reply:
That's the goal right?